Description: enables libunwind for gperftools
 libstacktrace, a component of gperftools, fails to compile on
 arm64 without libunwind.
Author: Vasily Smirnov <vasilii.smirnov@globallogic.com>
---
--- a/vendor/github.com/armPelionEdge/greasego/deps/src/greaseLib/deps/install-deps.sh
+++ b/vendor/github.com/armPelionEdge/greasego/deps/src/greaseLib/deps/install-deps.sh
@@ -39,7 +39,7 @@ cp Makefile.in.orig Makefile.in
 autoreconf -i
 #make clean
 echo "Echo building dependencies..." >> $LOG
-./configure $CONFIG_OPTIONS --prefix=${DEPS_DIR}/build --enable-frame-pointers --with-pic 2>&1 >> $LOG || echo "Failed in configure for gperftools" >> $LOG
+./configure $CONFIG_OPTIONS --prefix=${DEPS_DIR}/build --enable-libunwind --enable-frame-pointers --with-pic 2>&1 >> $LOG || echo "Failed in configure for gperftools" >> $LOG
 make -j4 2>&1 >> $LOG || echo "Failed to compile gperftools" >> $LOG
 make install 2>&1 >> $LOG || echo "Failed to install gperftools to: $DEPS_DIR/build" >> $LOG
 # does not copy libstacktrace for some reason
